addOns/automation/src/main/resources/org/zaproxy/addon/automation/resources/env.yaml

```diff
@@ -0,0 +1,14 @@
+--- # OWASP ZAP automation configuration file, for more details see https://www.zaproxy.com/docs/(TBA)
+env:                                   # The environment, mandatory
+  contexts :                           # List of 1 or more contexts, mandatory
+    - name: context 1                  # Name to be used to refer to this context in other jobs, mandatory
+      url:                             # The top level url, mandatory, everything under this will be included
```

Comment:
Not to change anything now but in anticipation of questions later. We've been asked before "what if my authentication is outside my context?"

Reply:
Good point! Will def need to cope with that case...